cmd/preset.go

```diff
@@ -27,29 +27,33 @@ var ErrPresetFilesAlreadyExists = errors.New("some preset files already exist")
 
 func init() {
 	var (
-		init    = NewKoolInit()
-		initCmd = NewInitCommand(init)
+		preset    = NewKoolPreset()
+		presetCmd = NewPresetCommand(preset)
 	)
 
-	rootCmd.AddCommand(initCmd)
+	rootCmd.AddCommand(presetCmd)
 }
 
-// NewKoolInit creates a new handler for init logic
-func NewKoolInit() *KoolInit {
-	return &KoolInit{
+// NewKoolPreset creates a new handler for preset logic
+func NewKoolPreset() *KoolPreset {
+	return &KoolPreset{
 		*newDefaultKoolService(),
-		&KoolInitFlags{false},
+		&KoolPresetFlags{false},
 		&presets.DefaultParser{Presets: presets.GetAll()},
 		shell.NewPromptSelect(),
 	}
 }
 
-// Execute runs the init logic with incoming arguments.
-func (i *KoolInit) Execute(args []string) (err error) {
-	var fileError, preset string
+// Execute runs the preset logic with incoming arguments.
+func (i *KoolPreset) Execute(args []string) (err error) {
+	var fileError, preset, language string
```

Comment:
right now this command pretty much NEEDS an TTY to execute. I guess would be nice to add a check for it, and error out with a proper message in case it is running under not-TTY environment.

Reply:
@fabriciojs it only needs TTY when no args are passed. Could we validate only for this situation?

Reply:
Yes! that sounds correct, only when needed to ask for the preset!